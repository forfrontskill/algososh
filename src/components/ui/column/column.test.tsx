
import renderer from 'react-test-renderer';
import { Column } from './column';

describe('Test Column component', () => {

    it('Column rendering', () => {
        const tree = renderer
            .create(<Column index={1} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});