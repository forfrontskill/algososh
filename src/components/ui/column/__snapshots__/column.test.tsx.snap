// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Column component Column rendering 1`] = `
<div
  className="content "
>
  <div
    className="column default"
    style={
      Object {
        "height": 3.2,
      }
    }
  />
  <p
    className="text text_type_column text_color_input mt-3"
  >
    1
  </p>
</div>
`;
